OpenRA.Mods.Common/Activities/Air/Fly.cs

```diff
@@ -53,7 +53,7 @@ public Fly(Actor self, Target t, WPos? initialTargetPosition = null, Color? targ
 		public static void FlyTick(Actor self, Aircraft aircraft, int desiredFacing, WDist desiredAltitude, WVec moveOverride, int turnSpeedOverride = -1)
 		{
 			var dat = self.World.Map.DistanceAboveTerrain(aircraft.CenterPosition);
-			var isSlider = aircraft.Info.FlightDynamics.HasFlag(FlightDynamic.Slide);
+			var isSlider = aircraft.Info.CanSlide;
```

Comment:
This is only used in one place and isn't much longer than the variable name, so please move inline.